Xamarin.Essentials/Sms/Sms.ios.cs

```diff
@@ -17,8 +18,8 @@ static Task PlatformComposeAsync(SmsMessage message)
             var messageController = new MFMessageComposeViewController();
             if (!string.IsNullOrWhiteSpace(message?.Body))
                 messageController.Body = message.Body;
-            if (!string.IsNullOrWhiteSpace(message?.Recipient))
-                messageController.Recipients = new[] { message.Recipient };
+
+            messageController.Recipients = message.Recipients.Where(x => !string.IsNullOrWhiteSpace(x)).ToArray();
```

Comment:
is there a reason we are filtering these? do we filter other platforms? at a high level this should be filtered in shared code.

Reply:
Moved filtering to shared code. I just continued the fashion in which they were checked before.